2005-11-01  Kazuhiro Inaoka <inaoka.kazuhiro@renesas.com>

	* sysdeps/unix/sysv/linux/i386/fcntl.c: Include "kernel-features.h"
	instead of "../kernel-features.h".
	(ports/sysdeps/unix/sysv/linux/m32r/fcntl.c include this file.)

Index: sysdeps/unix/sysv/linux/i386/fcntl.c
===================================================================
RCS file: /cvs/glibc/libc/sysdeps/unix/sysv/linux/i386/fcntl.c,v
retrieving revision 1.23
diff -u -p -r1.23 fcntl.c
--- sysdeps/unix/sysv/linux/i386/fcntl.c	16 Jul 2004 18:11:11 -0000	1.23
+++ sysdeps/unix/sysv/linux/i386/fcntl.c	9 May 2006 05:46:42 -0000
@@ -23,7 +23,7 @@
 #include <stdarg.h>
 
 #include <sys/syscall.h>
-#include "../kernel-features.h"
+#include "kernel-features.h"
 
 #if __ASSUME_FCNTL64 == 0
 /* This variable is shared with all files that check for fcntl64.  */
